infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationFSM.java

```diff
@@ -242,7 +237,6 @@ public LogReplicationFSM(CorfuRuntime runtime, SnapshotReader snapshotReader, Da
 
         this.snapshotReader = snapshotReader;
         this.logEntryReader = logEntryReader;
-        this.remoteCluster = remoteCluster;
```

Comment:
Can you explain why the class field is removed but `remoteCluster` is still passed to the constructor?

Reply:
Because its information was logged in this line -
```
log.info("Log Replication FSM initialized, streams to replicate {} to remote cluster {}",
                config.getStreamsToReplicate(), remoteCluster.getClusterId());
```                
I thought this log message could be useful and it has remote cluster information.